tests/rptest/services/producer_swarm.py

```diff
@@ -45,19 +45,29 @@ def __init__(self,
                  unique_topics: Optional[bool] = False,
                  messages_per_second_per_producer: Optional[int] = None):
         super(ProducerSwarm, self).__init__(context, num_nodes=1)
+
+        def assert_int(v: Any) -> int | None:
+            assert isinstance(v, int), f"not an int: {v}"
+            return v
+
+        def assert_int_or_none(v: Any) -> int | None:
+            assert v is None or isinstance(v, int), f"not an int or none: {v}"
+            return v
+
         self._redpanda = redpanda
         self._topic = topic
-        self._producers = producers
-        self._records_per_producer = records_per_producer
-        self._messages_per_second_per_producer = messages_per_second_per_producer
+        self._producers = assert_int(producers)
+        self._records_per_producer = assert_int(records_per_producer)
+        self._messages_per_second_per_producer = assert_int_or_none(
+            messages_per_second_per_producer)
         self._log_level = log_level
         self._properties = properties
-        self._timeout_ms = timeout_ms
+        self._timeout_ms = assert_int(timeout_ms)
```

Comment:
All the kgo services suffer from the same issue.

Reply:
:(